setup.py

```diff
@@ -106,7 +106,8 @@ def write_version(filename=os.path.join(*['airflow',
 ]
 celery = [
     'celery>=3.1.17',
-    'flower>=0.7.3'
+    'flower>=0.7.3',
+    'karellen-kombu-ext==4.0.2'
```

Comment:
I agree. I will work with the maintainers to Kombu to do whatever it take to get it in.